tutorial.ipynb

```diff
@@ -0,0 +1,665 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set parameters\n",
+    "freqs = list(range(1, 60 + 1))   # fequency values (Hz) over which to calculate TFR\n",
+    "times = np.arange(n_times) / samp_freq  # seconds\n",
+    "event_band = [15, 29]  # beta band (Hz)\n",
+    "thresh_FOM = 6.  # factor-of-the-median threshold"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Step 1**: **TIMESERIES** -> **TFR**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# calculate TFR\n",
+    "tfrs = se.tfr(subj_data, freqs, samp_freq)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Step 2**: **TFR** -> **SPECTRAL EVENTS**\n",
+    "\n",
+    "(Note that our band-of-interest, as guided by the literature and discussed in more detail in a few moments, is the **beta band (15-29 Hz)**.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# find spectral events!!\n",
+    "spec_events = se.find_events(tfr=tfrs, times=times, freqs=freqs,\n",
+    "                             event_band=event_band, threshold_FOM=thresh_FOM)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`spec_events` is is a nested list (trials x events). Each event is annotated by\n",
+    "dictionary key-value pairs that include `Peak Time`, `Peak Frequency`,\n",
+    "`Peak Power`, `Duration`, `Frequency Span`, amoung a few others."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "200\n",
+      "1\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'Peak Frequency': 25,\n",
+       " 'Lower Frequency Bound': 20,\n",
+       " 'Upper Frequency Bound': 30,\n",
+       " 'Frequency Span': 10,\n",
+       " 'Peak Time': 0.82,\n",
+       " 'Event Onset Time': 0.7733333333333333,\n",
+       " 'Event Offset Time': 0.9,\n",
+       " 'Event Duration': 0.1266666692,\n",
+       " 'Peak Power': 1.677525973282318e-16,\n",
+       " 'Normalized Peak Power': 10.087765786470593}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "print(len(spec_events))  # number of trials\n",
+    "print(len(spec_events[0]))  # number of events in the 1st trial\n",
+    "spec_events[0][0]  # features of the 1st event of the 1st trial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before we investigate anything else about the events that were detected in the\n",
+    "the data (we'll come back to this shortly), let's see what the average\n",
```

Comment:
Reading through it does seem a little funny that we run this event detection, and then don't use it for a large chunk of the tutorial (I had to scroll back to remind myself where it was actually created).

With very little changes, I think you can just move this below the first plot where you show single trial spectrograms. That way it's really obvious what spectral events are visually, and then we've got a handy function to detect them automatically as well.

Reply:
fixed in https://github.com/jonescompneurolab/SpectralEvents/pull/9/commits/dba5d45443e3cd42b6afe9b393ad6fa65fbb9a79